### PR TITLE
Luke/gauge create class conversion

### PR DESCRIPTION
**Pull request by lukeschunk**
createClass conversion in `Gauge.js` and `GaugeDocs.js`. 

Also noticed the header in the GaugeDocs had a typo so I fixed that.

### CODE REVIEW

File: docs/components/GaugeDocs.js [hunk 1]
```diff
@@ -4,13 +4,13 @@ const { Gauge } = require('mx-react-components');
...
 
-const GaugeDocs = React.createClass({
+class GaugeDocs extends React.Component {
   render () {
     return (
       <div>
         <h1>
           Gauge
-          <label>A D3 gauge that supports a multiple multiple, and reference dots.</label>
+          <label>A D3 gauge that supports multiple colors for each tranche of the gauge, and reference dots.</label>
```
Comment (paniclater) [inline docs/components/GaugeDocs.js]: tranche, fancy! 🍪

**Comment by paniclater** [comment]
:shipit:

File: docs/components/GaugeDocs.js (hunk 1 above)
Comment (lukeschunk) [inline docs/components/GaugeDocs.js]: hahah @vintagepenguin vocab there

**Comment by paniclater** [comment]
:shipit:

**Pull request merged**